extensions/amp-story/1.0/amp-story.css

```diff
@@ -205,9 +206,9 @@ amp-story-page[active] {
   max-width: 280px !important;
   min-width: 280px !important;
   width: 280px !important;
-  transform: translate3d(100%, 0, 0) !important;
-  transition: transform 0.2s cubic-bezier(0.4, 0.0, 1, 1) !important;
-  display: block !important;
+  animation-timing-function: cubic-bezier(0.4, 0.0, 1, 1)  !important;
```

Comment:
nit: material guideline recommends using the sharp curve for items temporarily leaving the screen

Reply:
It's a bit tricky, both are recommended for elements leaving the screen, depending on the context... We mostly use this acceleration curve in Stories tho